microshift_release_notes/microshift-4-20-release-notes.adoc

```diff
@@ -0,0 +1,165 @@
+
+//link:https://access.redhat.com/support/offerings/techpreview[Technology Preview Features Support Scope]
+
+//[id="microshift-4-20-ai_{context}"]
+//=== Red{nbsp}Hat OpenShift AI with {microshift-short}
+
+//[id="microshift-4-20-rhoai-ga_{context}"]
+//==== Serve your trained models on edge deployments
+
+//With this release, a streamlined version of {rhoai-full} {rhoai} is Generally Available. {rhoai} can be used to serve artificial intelligence or machine learning (AI/ML) models on {microshift-short}. Develop and train your AI models in the data center or cloud, then deploy them to the edge where these models can make decisions without a human user. For more information, see xref:../microshift_ai/microshift-rhoai.adoc#microshift-rhoai[Using Red{nbsp}Hat OpenShift AI with {microshift-short}].
+
+//[id="microshift-4-20-deprecated-and-removed-features_{context}"]
+//== Deprecated and removed features
+
+//Some features available in previous releases have been deprecated or removed. Deprecated functionality is still included in and continues to be supported, but will be removed in a future release. Deprecated features and functionality are not recommended for new deployments.
+
+//[id="microshift-4-20-x-deprecated_{context}"]
+//=== placeholder
+
+
+//NOTE: Do NOT repeat bug fixes already noted in previous version z-streams
+//[id="microshift-4-20-bug-fixes_{context}"]
+//== Bug fixes
+
+//*
+
+[id="microshift-4-20-known-issues_{context}"]
+== Known issues
+
+* The maximum transmission unit (MTU) value in {microshift-short} OVN-K overlay networking must be 100 bytes smaller than the MTU value of the base network. {microshift-short} automatically configures the value using the MTU value of the default gateway of the host. If the auto-configuration does not work correctly, the MTU value must be configured manually. For more information, see xref:../microshift_networking/microshift-cni.adoc#microshift-network-topology_microshift-about-ovn-k-plugin[Network topology].
+
+[id="microshift-4-20-additional-release-notes_{context}"]
+== Additional release notes
+Release notes for related components and products are available in the following documentation:
+
+[NOTE]
+====
+The following release notes are for downstream Red{nbsp}Hat products only; upstream or community release notes for related products are not included.
+====
+
+//TODO check the latest gitops release version
+[id="microshift-4-20-additional-release-notes-gitops_{context}"]
+=== GitOps release notes
+See link:https://docs.redhat.com/en/documentation/red_hat_openshift_gitops/latest/html/release_notes/index[Red{nbsp}Hat OpenShift GitOps: Highlights of what is new and what has changed with this OpenShift GitOps release] for more information. You can also go to the Red{nbsp}Hat package download page and search for "gitops" if you just need the latest package, link:https://access.redhat.com/downloads/content/package-browser[Red{nbsp}Hat packages].
```

Comment:
```suggestion
See link:https://docs.redhat.com/en/documentation/red_hat_openshift_gitops/latest/html/release_notes/index[Red{nbsp}Hat OpenShift GitOps: Highlights of what is new and what has changed with this OpenShift GitOps release] for more information. You can also go to the link:https://access.redhat.com/downloads/content/package-browser[Red{nbsp}Hat package download page] and search for "gitops" if you just need the latest package.
```
I'm sure you're just copying this text over from the previous release notes, but I saw this and wondered if the linked text should be moved.